R/SurveyData.R

```diff
@@ -24,7 +24,8 @@
 #'   responses = list(
 #'     age = levels(shape_survey$age),
 #'     gender = levels(shape_survey$gender),
-#'     vote_for = levels(shape_survey$vote_for),
+#'     vote_for = data.frame(data = levels(shape_survey$vote_for),
```

Comment:
can you add a comment indicating that we're using the option of a data frame because the values in the data are different than the actual responses?

Reply:
Yup good suggestiob